src/common/utils/market-utils.ts

```diff
@@ -0,0 +1,19 @@
+import { useQuery } from 'react-query';
```

Comment:
Great that you extracted all the logic. The current folder organisation is not great, but you could maybe place this file under `common/hooks/` instead.